test_resources/test_data/rows_categories.edn

```diff
@@ -0,0 +1,75 @@
+[1 "African"]
```

Comment:
Test data schemas now includes `id` explictly and rows now include it was well, meaning we don't have to goof around trying to figure out the equivalent of `SERIAL`/`AUTO_INCREMENT`/whatever for every single database we support. More complexity eliminated and things sped up even more